### PR TITLE
Remove voyager usecases from Integration Tests

### PR DESCRIPTION
**Pull request by anpanigr**
Removed all the Voyager Integration tests  from Operator Integration tests. 
Here I am not removing the Voyager Reference form LoadBalalcerUtil class.  
So the infrastructure/util code is still there in case we need in future.

External Jenkin Run
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/6598/
https://build.weblogick8s.org:8443/job/wko-okd-test/26/

### CODE REVIEW

**Review by maggiehe00 — CHANGES_REQUESTED** [review]
The change should be OK on OKD but verifying the change on OKD wko-okd-wls-cert profile would be safer especially for ItManagedCoherence.

**Review by maggiehe00 — APPROVED** [review]
wko-okd run is good :)

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManagedCoherence.java [hunk 1]
```diff
@@ -168,40 +161,36 @@ void testMultiClusterCoherenceDomain() {
...
           -> coherenceCacheTest(cluster1IngressHost), "Test Coherence cache failed");
       assertTrue(testCompletedSuccessfully, "Test Coherence cache failed");
     } else {
 
-      // create Voyager ingress resource
       Map<String, Integer> clusterNameMsPortMap = new HashMap<>();
       for (int i = 1; i <= NUMBER_OF_CLUSTERS; i++) {
         clusterNameMsPortMap.put(CLUSTER_NAME_PREFIX + i, MANAGED_SERVER_PORT);
       }
+      // clusterNameMsPortMap.put(clusterName, managedServerPort);
```
Comment (vanajamukkara) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManagedCoherence.java:175]: remove commented code

**Review by vanajamukkara — APPROVED** [review]
LGTM

**Pull request merged**